prow/cluster/cluster.yaml

```diff
@@ -483,6 +487,8 @@ spec:
         - --config-path=/etc/config/config.yaml
         - --job-config-path=/etc/job-config
         - --dry-run=false
+        - --github-endpoint=http://ghproxy
```

Comment:
Can we first deploy it and then make things use it?

Reply:
Sure, I'll split it up. I was going to do separate PRs originally, but the GH client will fallback to using the API directly if ghproxy isn't working for any reason so I didn't think it was very risky.

Reply:
In that case might be fine to just do it but either way

Reply:
Done.
Follow up: https://github.com/GoogleCloudPlatform/oss-test-infra/pull/345